pkg/neorpc/errors.go

```diff
@@ -30,33 +30,95 @@ const (
 const (
 	// RPCErrorCode is returned on RPC request processing error.
 	RPCErrorCode = -100
+	// RPCErrorUnknownBlockCode is returned on RPC request processing error: Unknown block
+	RPCErrorUnknownBlockCode = -101
+	// RPCErrorUnknownContractCode is returned on RPC request processing error: Unknown contract
+	RPCErrorUnknownContractCode = -102
+	// RPCErrorUnknownTransactionCode is returned on RPC request processing error: Unknown transaction
+	RPCErrorUnknownTransactionCode = -103
+	// RPCErrorUnknownStorageItemCode is returned on RPC request processing error: Unknown storage item
+	RPCErrorUnknownStorageItemCode = -104
+	// RPCErrorUnknownScriptContainerCode is returned on RPC request processing error: Unknown script container
+	RPCErrorUnknownScriptContainerCode = -105
+	// RPCErrorUnknownStateRootCode is returned on RPC request processing error: Unknown state root
+	RPCErrorUnknownStateRootCode = -106
+	// RPCErrorUnknownHeaderCode is returned on RPC request processing error: Unknown header
+	RPCErrorUnknownHeaderCode = -107
 )
 
 var (
 	// ErrInvalidParams represents a generic 'invalid parameters' error.
 	ErrInvalidParams = NewInvalidParamsError("invalid params")
+
 	// ErrUnknownBlock is returned if requested block is not found.
-	ErrUnknownBlock = NewError(RPCErrorCode, "Unknown block", "")
+	ErrUnknownBlock = NewError(RPCErrorUnknownBlockCode, "Unknown block", "")
+	// ErrUnknownContract is returned when requested unknown contract
+	ErrUnknownContract = NewError(RPCErrorUnknownContractCode, "Unknown contract", "")
 	// ErrUnknownTransaction is returned if requested transaction is not found.
-	ErrUnknownTransaction = NewError(RPCErrorCode, "Unknown transaction", "")
-	// ErrUnknownHeader is returned when requested header is not found.
-	ErrUnknownHeader = NewError(RPCErrorCode, "Unknown header", "")
+	ErrUnknownTransaction = NewError(RPCErrorUnknownTransactionCode, "Unknown transaction", "")
+	// ErrUnknownStorageItem is returned when requested unknown storage item
+	ErrUnknownStorageItem = NewError(RPCErrorUnknownStorageItemCode, "Unknown storage item", "")
 	// ErrUnknownScriptContainer is returned when requested block or transaction is not found.
-	ErrUnknownScriptContainer = NewError(RPCErrorCode, "Unknown script container", "")
+	ErrUnknownScriptContainer = NewError(RPCErrorUnknownScriptContainerCode, "Unknown script container", "")
 	// ErrUnknownStateRoot is returned when requested state root is not found.
-	ErrUnknownStateRoot = NewError(RPCErrorCode, "Unknown state root", "")
+	ErrUnknownStateRoot = NewError(RPCErrorUnknownStateRootCode, "Unknown state root", "")
+	// ErrUnknownHeader is returned when requested header is not found.
+	ErrUnknownHeader = NewError(RPCErrorUnknownHeaderCode, "Unknown header", "")
+
+	// ErrInvalidHeight represents SubmitError with code -201.
+	ErrInvalidHeight = NewSubmitError(-201, "Invalid height")
```

Comment:
I don't see it in https://github.com/neo-project/proposals/blob/b20970e325e7ed8ab3895bb99e6c230574999dd2/nep-Y.mediawiki

Reply:
Hm, I forgot we have these errors in the proposal. But -200... codes were in the original proposal (https://github.com/neo-project/neo-modules/issues/728#issue-1267784057). Are they removed intentionally?

Reply:
And you've also mentioned a new `-203` code [here](https://github.com/neo-project/neo-modules/issues/728#issuecomment-1190761261), and it's missing from the final proposal.

Reply:
Well, it's an improved version!

Reply:
Good, then it's my fault I, completely forgot about the final proposal. We have to follow the https://github.com/neo-project/proposals/pull/156.

Reply:
I used this https://github.com/neo-project/neo-modules/issues/728